appserver/security/webservices.security/src/main/java/com/sun/enterprise/security/webservices/server/ServerSecurityPipe.java

```diff
@@ -157,6 +157,7 @@ private Packet processRequest(Packet request) throws Exception {
             authenticationService.auditInvocation(validatedRequest, status);
         }
 
+        _logger.log(FINE, "Validation status: {0}", status);
```

Comment:
I moved the log and fixed the log message - the original message key did not exist.